src/app/@ansyn/map-facade/components/imagery-rotation/imagery-rotation.component.ts

```diff
@@ -73,6 +71,10 @@ export class ImageryRotationComponent {
 				protected store: Store<any>) {
 	}
 
+	toDegrees(radians: number): number {
```

Comment:
you can move this method to "imagery
ol and mapFacade knows it

Reply:
just on line..